src/Bicep.LangServer/Snippets/SnippetsProvider.cs

```diff
@@ -360,5 +357,22 @@ private Snippet GetEmptySnippet()
 
             return new Snippet("{\n\t$0\n}", CompletionPriority.Medium, label, label);
         }
+
+        public IEnumerable<Snippet> GetModuleBodyCompletionSnippets(TypeSymbol typeSymbol)
+        {
+            yield return GetEmptySnippet();
+
+            if (typeSymbol is ModuleType moduleType && moduleType.Body is ObjectType objectType)
```

Comment:
@majastrz , I investigated DiscriminatedObjectType usage and I believe it's not applicable to modules. Correct me if I am wrong.

Reply:
Yup, modules only use ObjectType.